src/Stripe.net/Entities/SubscriptionSchedules/SubscriptionSchedulePhase.cs

```diff
@@ -102,56 +64,29 @@ public PaymentMethod DefaultPaymentMethod
         internal ExpandableField<PaymentMethod> InternalDefaultPaymentMethod { get; set; }
         #endregion
 
-        /// <summary>
-        /// The default tax rates which apply to the phase of this subscription schedule.
-        /// </summary>
         [JsonProperty("default_tax_rates")]
         public List<TaxRate> DefaultTaxRates { get; set; }
 
-        /// <summary>
-        /// The end of this phase of the subscription schedule.
-        /// </summary>
         [JsonProperty("end_date")]
         [JsonConverter(typeof(UnixDateTimeConverter))]
         public DateTime EndDate { get; set; }
 
-        /// <summary>
-        /// The default invoice settings for this phase.
-        /// </summary>
         [JsonProperty("invoice_settings")]
-        public SubscriptionScheduleInvoiceSettings InvoiceSettings { get; set; }
+        public SubscriptionSchedulePhaseInvoiceSettings InvoiceSettings { get; set; }
 
-        /// <summary>
-        /// Plans to subscribe during this phase of the subscription schedule.
-        /// </summary>
         [JsonProperty("plans")]
-        public List<SubscriptionSchedulePhaseItem> Plans { get; set; }
+        public List<SubscriptionSchedulePhasePlan> Plans { get; set; }
```

Comment:
Argl, that one is annoying, we know it will be renamed *soon*. How hard is it to special-case that one?